nimble/include/nimble/ble.h

```diff
@@ -109,8 +109,8 @@ struct ble_mbuf_hdr_rxinfo
 struct ble_mbuf_hdr_txinfo
 {
     uint8_t flags;
-    uint8_t offset;
-    uint8_t pyld_len;
+    uint16_t offset;
+    uint16_t pyld_len;
     uint8_t hdr_byte;
```

Comment:
move `hdr_byte` after `flags` to optimize alignment in struct

Reply:
Done